step.yml

```diff
@@ -261,7 +261,8 @@ inputs:
       category: Debug
       title: "Should retry test on failure?"
       description: |-
-        If `should_retry_test_on_fail: yes` step will retry the test if first attempt failed.
+        If you set this input to `yes`, the Step will rerun only the failed test case without having to rerun all the test cases. 
+        Please note that this feature is only available from Xcode 13 and above, earlier versions support rerunning ALL test cases instead of a specific failed one.  
```

Comment:
I think this can be misleading for 2 reasons:
1. "[...] the Step will rerun only the failed test case without having to rerun all the test cases." -> This is true, but not in this context. The step will rerun only the failed tests because it runs on stacks that support Xcode 13 or above, not because this switch is turned on.
2. We don't have another switch nor a "default behavior" that reruns all tests on failure.

I'd suggest rewording it, so that it articulates that:
1. This option allows you to retry your failed tests.
2. With Xcode 13 and above, it will retry only your failed tests. Below that, it will rerun all of your tests.

Wdyt? Does that make sense?

Reply:
```suggestion
```

Reply:
I've added my suggestion above; the "please note" part can be deleted.